lib/fastlane/plugin/versioning/actions/get_version_number_from_git_branch.rb

```diff
@@ -5,7 +5,7 @@ def self.run(params)
         if Helper.test?
           branch = 'releases/release-1.3.5'
         else
-          branch = Actions.git_branch
+          branch = other_action.git_branch
```

Comment:
forgive me for my ignorance, but is there any difference between this invocation and invoking the action by its classname? 

i.e. `GitBranchAction.run` 

the only one I could guess at is that the Params you pass to `get_version_number_from_git_branch` probably don't really align with any that `git_branch` expects. maybe.